File: voting_mesin.java
import java.util.Scanner;

public class voting_mesin {     
public static void main(String[] args) {

    Scanner sc = new Scanner(System.in);
    Details D = new Details(sc);
    VoteSystem v = new VoteSystem(sc,D);
    boolean Loop = true;
    
    while (Loop) {

    System.out.println("ENTER 1 IS EDIT THE DETAILS || ENTER 2 IS START VOTING ");
    int t = sc.nextInt();
    sc.nextLine();
    switch (t) {
        case 1:
            D.INPUT();
            break;
        case 2:
        if(D.np <=0){
            System.out.println("PLEAS FILL THE DETAILS FIRST...!");
        }
        else{
            v.vote();
            Loop = false;
        }
            break;
        default: 
            Loop = false;
            break;
   }
  }
 }
}

class Details {

     Scanner sc;
     Details(Scanner scanner){
      this.sc = scanner;
     }
    int wn,np;
    String area,head;
    void INPUT(){
    System.out.print("ENTER THE WARD NUMBER : ");
    wn = sc.nextInt();
    sc.nextLine();
    System.out.print("ENTER THE AREA NAME : ");
    area = sc.nextLine();
    System.out.print("ENTER THE NUMBER OF PEOPLE :");
    np = sc.nextInt();
    sc.nextLine();
    System.out.print("ENTER THE NAME HEAD OF THE VOTING DEPARTMENT : ");
    head = sc.nextLine();
  }  
}

class VoteSystem {

    Details D;
    Scanner sc;
    VoteSystem ( Scanner scanner , Details details){
          this.sc = scanner;
          this.D = details;
    }
    char c ;
    int count ;
    int muss,hit,fran,kim;
    boolean F = true;
    
void vote(){
    for(int i = 1;i<=D.np;i++){  
      count = i;
      v_input();
    }
    Result();
}

void v_input(){
    System.out.println("\nVOTING PAGE\n");
    System.out.println("\nNUMBER"+count);
    System.out.println("\nMUSSOLINI PRESS 1\nHITLER PRESS 2\nFRANCISCO FRANCO PRESS 3\nKIM JONG UN PRESS 4");
    c = sc.nextLine().charAt(0);
    
    switch (c) {
        case '1' :
            muss++;
            System.out.println(" THANKS FOR VOTEING TO MUSSOLINI ");
            break;
     
        case '2' :
            hit++;
            System.out.println(" THANKS FOR VOTEING TO HITLER ");
            break;

        case '3' :
            fran++;
            System.out.println(" THANKS FOR VOTEING TO FRANCISCO FRANCO ");
            break;

        case '4' :
            kim++;
            System.out.println(" THANKS FOR VOTEING TO KIM JONG UN ");
            break;

        default:
            System.out.println(" SOMETHING WAS WRONG ");
            break;
        } 
}

void Result(){
      System.out.println("\nRESULT\n");
      System.out.println("HEAD NAME : "+D.head);
      System.out.println("AREA : "+D.area);
      System.out.println("WARD NUMBER : "+D.wn+"\n");
      System.out.println("MUSSOLINI : "+muss+" VOTE");
      System.out.println("HITLER : "+hit+" VOTE");
      System.out.println("FRANCISCO FRANCO : "+fran+" VOTE");
      System.out.println("KAVINKIM JONG UN : "+kim+" VOTE");
}
}

File: voting_mesin.cpp
#include <iostream>
using namespace std;

class Details{
public:
   int wn,np;
   string area,head;
   
   void Input(){
    cout << "ENTER THE WARD NUMBER : ";
    cin >> wn;
    cout <<"ENTER THE AREA NAME : ";
    cin >> area;
    cout << "ENTER THE NUMBER OF PEOPLES : ";
    cin >> np;
    cout << "ENTER THE NAME OF THE HEAD OF ELECTION COMMISSION : ";
    cin >> head;
   }
};
class Voting_System{
private:

    Details D;
    int count = 0;
    int muss = 0;int hit = 0;int kim =  0;int fran = 0;
    char T;

public:
   
    Voting_System(Details details) : D (details){}
    void Vote(){
       for (int i = 1; i < D.np; i++)
       {
          count++;
          V_input();
       }
       Result();   
    }

    void V_input(){
        cout<<"\nVOTING PAGE\n";
        cout<<"COUNT : "<<count;
        cout<<"\n| MUSSOLINI PRESS 1 | HITLER PRESS 2 | KIM JONG UN PRESS 3 | FRANCISCO FRANCO PRESS 4 |\n";
        cin>>T;
        switch (T) {
        case '1' :
            muss++;
            cout<<" THANKS FOR VOTEING ";
            break;
     
        case '2' :
            hit++;
            cout<<" THANKS FOR VOTEING ";
            break;

        case '4' :
            fran++;
            cout<<" THANKS FOR VOTEING ";
            break;

        case '3' :
            kim++;
            cout<<" THANKS FOR VOTEING ";
            break;

        default:
            cout<<" SOMETHING WAS WRONG ";
            break;
        } 
    }

    void Result(){ 
       cout<<"\nRESULT\n";
       cout<<"WARD NUMBER : "<<D.wn<<"\nAREA NAME : "<<D.area<<"\nNUMBER OF PEOPLE IN THIS WARD : "<<D.np<<"\nHEAD OF THE VOTE COMMISSIONER NAME : "<<D.head;
       cout<<"\n MUSSOLINI "<<muss<<"\n HITLER "<<hit<<"\n KIM JONG UN "<<kim<<"\n FRANCISCO FRANCO "<<fran;
    }
};

int main(){
        Details d;
        Voting_System v(d);
        char n;
        bool L = true;

        while(L){
        cout<<"\n| PRESS 1 IS ENTER DETAILS | PRESS 2 IS START THE VOTING |\n";
        cin>>n;
        
        switch(n){
           case '1':
           d.Input();
           break;

           case '2':
           if(d.np <=0){
            cout<<"FILL THE DETAILS FIRST SIR...!";
            L=true;
           }
           else{
           v.Vote();
           }
           default:
           L= false;
           break;
        }
        }
};
